browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -232,6 +232,67 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_RESOLVE_METHOD_ETHEREUM" desc="Text for resolve method Ethereum.">
         Ethereum
       </message>
+      <message name="IDS_BRAVE_NEWS_OPTIN_TITLE" desc="Text for opt in card title">
+        Today’s top stories in a completely private feed, just for you.
+      </message>
+
+       <message name="IDS_BRAVE_NEWS_OPTIN_SUBTITLE" desc="Text for opt in card subtitle">
+        Brave News is ad-supported with completely private and anonymized ads matched on your device.
+      </message>
+
+      <message name="IDS_BRAVE_NEWS_AD" desc="Text for Brave ad logo">
+        Ad
+      </message>
+      <message name="IDS_BRAVE_NEWS_OPTIN_BUTTON" desc="Text for opt in card subtitle">
+        Turn on Brave News
+      </message>
+      <message name="IDS_BRAVE_NEWS_OPTIN_LEARN_MORE" desc="Text for opt in card subtitle">
+        Learn more about your data
+      </message>
+      <message name="IDS_BRAVE_NEWS_OPEN_NEW_TAB" desc="Text for opening the link in a new tab option for the long press on title action for Brave news">
+        Open in New Tab
+      </message>
+      <message name="IDS_BRAVE_NEWS_OPEN_NEW_PRIVATE_TAB" desc="Text for opening the link in a new private tab option for the long press on title action for Brave news">
+        Open in New Private Tab
+      </message>
+      <message name="IDS_BRAVE_NEWS_DISABLE_CONTENT" desc="Text for disable content option for the long press on title action">
+        Disable content from
```

Comment:
This needs value replacement - the value won't always be at the end of the string in different languages.